locales/en-US/strings.properties

```diff
@@ -117,3 +117,11 @@ pocket_feedback_header=The best of the web, curated by over 25 million people.
 # (pocket_feedback_header) to provide more information about Pocket.
 pocket_feedback_body=Pocket, a part of the Mozilla family, will help connect you to high-quality content that you may not have found otherwise.
 pocket_send_feedback=Send Feedback
+
+# LOCALIZATION NOTE (migrate_*): This is shown in the migration message
+migrate_manual_import_msg=No problem. We can start you off with a few popular sites instead. You can always
+# LOCALIZATION NOTE (migrate_manual_import_link_msg): This is shown as the link in message
+migrate_manual_import_link_msg=import your information
+migrate_manual_import_msg_trail=later.
```

Comment:
I see that flod says it won't be difficult to localize, so maybe we just need to fix the implicit concatenation issue with {replacement}. Although actually given that we have a before-link and after-link text, that should provide localizers with enough flexibility to work with? @flodolo